studio-docs/source/metrics/field-usage.mdx

```diff
@@ -212,8 +236,31 @@ new ApolloServer({
 });
 ```
 
-If you do this, field execution statistics will not appear on the Fields page.
+If you do this, field execution statistics do not appear on the Fields page.
+
+#### Fractional sampling
+
+You can enable field-level instrumentation for a fixed fraction of all requests by passing a number between `0` and `1` as the `fieldLevelInstrumentation` option:
+
+```ts
+new ApolloServer({
+  plugins: [
+    ApolloServerPluginUsageReporting({
+      fieldLevelInstrumentation: 0.01,
+    }),
+  ],
+  // ...
+});
+```
+
+Make sure to pass a _number_ (like `0.01`), _not_ a function _returning_ a number (like `() => 0.01`). If you pass a number, Apollo Server randomly chooses to enable field-level instrumentation for each request according to the given probability.
```

Comment:
Oh this was my fault but it should probably say `_not_ a function _returning_ a constant number`. The point being that it's valid to write functions that sometimes return numbers and sometimes don't, but a function that always returns the same non-1 number means "always instrument and always multiply by this value".